src/metrocs/redistricting/StarGeneratorTest.java

```diff
@@ -12,27 +12,34 @@
 
 class StarGeneratorTest{
 
-    /* Verifies that locations inside pattern generated are not empty */
+    /** Verifies that locations inside pattern generated are not empty */
     @Test
     public void StarGeneratorEmptyTest(){
         ArrayList<ArrayList<Location>> sp = StarGenerator.generatePattern(3);
         assertFalse(sp.isEmpty());
     }
 
-    /* Verifies that the Star Generator method is not randomly generating a pattern */
+    /** Verifies that the Star Generator method is not randomly generating a pattern */
```

Comment:
Is there a requirement that the patterns returned by StarGenerator are presented in a specific order? The inherited "equals" method of ArrayList (from AbstractList) requires both lists to have the same elements in the same order.